StripeCore/StripeCoreTests/Categories/UIImage+StripeCoreTests.swift

```diff
@@ -1,3 +1,4 @@
+@_spi(STP)@testable import StripeCore
```

Comment:
This is weird, can we fix this somehow? Does `@testable @_spi(STP) import StripeCore` work for the formatter?

Reply:
Nope, with all variants it still ends up putting them together. Filed a [bug](https://github.com/apple/swift-format/issues/445) for it.
I'm making swift-format ignore it.
Moving it to the top of the file was wrong, I think it did that because there was no line break between the header and the first import, I fixed that.